pkg/image/sif.go

```diff
@@ -130,8 +143,20 @@ func (f *sifFormat) initializer(img *Image, fileinfo os.FileInfo) error {
 			switch fstype {
```

Comment:
Can't this code be merged with the above? Some function returning the FS type or an error?

Reply:
Yes it could, just for now does it work as expected for you ?

Reply:
No. I still get the same indeterminate error on inspect and mount failure on shell/run/exec

Reply:
Well it's weird that the cluster kernel doesn't support the default compression which is gzip, when image is compressed with another compression algorithm you get an info message like `squashfs image was compressed with <comp>, if it failed to run, please contact image's author`.

I think squashfs filesystem is supported by your kernel (just in case check with `grep squashfs /proc/filesystems`), otherwise you should get a `no such device` error if it was the case.

From `mount` man, the only corresponding error for this issue is `EINVAL source had an invalid superblock`